kinfer/protos/kinfer.proto

```diff
@@ -154,18 +173,34 @@ message IMUValue {
     IMUMagnetometerValue magnetic_field = 3;
 }
 
+message AngularVelocitySchema {
+}
+
+message AngularVelocityValue {
+    float x = 1;
+    float y = 2;
+    float z = 3;
+}
+
+message EulerRotationSchema {
+}
+
+message EulerRotationValue {
+    float x = 1;
+    float y = 2;
+    float z = 3;
+}
+
 /* --------- *
  * Timestamp *
  * --------- */
 
 message TimestampSchema {
-    int64 start_seconds = 1;
-    int32 start_nanos = 2;
+    float start_seconds = 1;
```

Comment:
we should just convert from the true timestamp to a float in the model rather than passing in a float. we want to have pretty precise timing